patroni/log.py

```diff
@@ -99,6 +106,8 @@ def reload_config(self, config):
                 self._queue_handler.queue.maxsize = config.get('max_queue_size', self.DEFAULT_MAX_QUEUE_SIZE)
 
             self._root_logger.setLevel(config.get('level', PatroniLogger.DEFAULT_LEVEL))
+            if config.get('traceback_level', PatroniLogger.TRACEBACK_LEVEL).lower() == 'debug':
```

Comment:
The `traceback_level` could be changed in the runtime.

Reply:
made it switch back to original behavior if configuration is changed back.